src/language-css/print/comma-separated-value-group.js

```diff
@@ -400,6 +407,12 @@ function printCommaSeparatedValueGroup(path, options, print) {
       continue;
     }
 
+    // don't print line when the next node is a comment and last node
+    // it will be printed with the comment in a  linesuffix
```

Comment:
```suggestion
    // it will be printed with the comment in a line suffix
```
should resolve the CI issue